src/3rdparty/monocypher/monocypher.cpp

```diff
@@ -66,8 +66,8 @@ namespace MONOCYPHER_CPP_NAMESPACE {
 #define ZERO(buf, size)            FOR(_i_, 0, size) (buf)[_i_] = 0
 #define WIPE_CTX(ctx)              crypto_wipe(ctx   , sizeof(*(ctx)))
 #define WIPE_BUFFER(buffer)        crypto_wipe(buffer, sizeof(buffer))
-#define MIN(a, b)                  ((a) <= (b) ? (a) : (b))
-#define MAX(a, b)                  ((a) >= (b) ? (a) : (b))
+#define MC_MIN(a, b)               ((a) <= (b) ? (a) : (b))
+#define MC_MAX(a, b)               ((a) >= (b) ? (a) : (b))
```

Comment:
Turns out, this is only because of our precompiled headers. Normally this wouldn't be an issue. So yeah, we have to fix it here, and it doesn't belong upstream :)